src/helpers/ms-graph.js

```diff
@@ -49,6 +49,7 @@ const msGraphPlugin = {
         baseUrl: azureClientBaseUrl,
         ...(proxy && {
           fetchOptions: {
+            // @ts-expect-error This is not typed
```

Comment:
would be worth adding this link in here - https://github.com/microsoftgraph/msgraph-sdk-javascript/issues/1646#issuecomment-2200783014